### PR TITLE
[ONCEHUB-63483] - [App2] [Licenses] : When the user name is entered in the search text field entered username is not able to select the checkbox for the second click of the enter button

### PR DESCRIPTION
**Pull request by sharmaaayush-OH**
### What does this PR do?
Improves accessibility of select dropdown.

### Why do we want to do that?
To fix known bugs and make select dropdown more keyboard accessible

### What are the high level changes?
Highlight the first element instead of first selected element on dropdown open.
Highlight an element in all cases if search filter returns at least one option to select from.

### What other information should the reviewer be aware of when looking at this code?
The relevant ticket is [ONCEHUB-63483](https://scheduleonce.atlassian.net/browse/ONCEHUB-63483)

### Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

[ONCEHUB-63483]: https://scheduleonce.atlassian.net/browse/ONCEHUB-63483?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by oh-integrations** [comment]
@sharmaaayush-OH: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | ea4cfccead394cd5f629966a9087953ab7b7abdf | [link](http://jenkinsx.onceplatform.com/namespaces/jx/pipelineruns/scheduleonce-once-ui-pr-1026-pr-hk6wl) | `/test`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "core-js": "3.26.1",
     "rxjs": "7.5.7",
     "tslib": "2.4.1",
-    "zone.js": "0.11.4"
+    "zone.js": "0.12.0"
```
Comment (Areeb2512) [inline package.json:42]: Downgrade it to "zone.js": "0.11.4"

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Areeb2512 — DISMISSED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[Areeb2512](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343048900 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [7.0.24] - 2023-03-16
+
+- [ONCEHUB-63483](https://scheduleonce.atlassian.net/browse/ONCEHUB-63483) When the user name is entered in the search text field entered username is not able to select the checkbox for the second click of the enter button.
+
 ## [7.0.22] - 2023-01-30
```
Comment (Areeb2512) [inline CHANGELOG.md:12]: Please make it 7.0.23 and date 2023-03-13

**Review by naveenoncehub — APPROVED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343087476 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343087476 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343087476 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#issuecomment-1471477416 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#issuecomment-1471477416 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#issuecomment-1471477416 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by naveenoncehub — APPROVED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343117971 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343117971 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-ashusrivastava**
You can assign the PR to them by writing `/assign @so-ashusrivastava` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by so-kaushal — APPROVED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343117971 "Approved")*, *[so-kaushal](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343166060 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)~~ [so-kaushal]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[naveenoncehub](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343117971 "Approved")*, *[so-kaushal](https://github.com/scheduleonce/once-ui/pull/1026#pullrequestreview-1343166060 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fonce-ui).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/once-ui/blob/qa/OWNERS)~~ [so-kaushal]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**